index.html

```diff
@@ -255,6 +255,25 @@ <h1>
 
         </section>
 
+        <div id="contact-section-container">
+            <section class="contact-section flex">
+                <h1 id="contact-header">Contact Me</h1>
+                <p id="contact-detail">If you have an application you are
+                    interested
+                    in developing,
+                    a feature that you need built or a project that needs
+                    coding.
+                    I’d love to help with it
+                </p>
+                <form action="https://formspree.io/f/mzbykgne" method="POST">
+                    <input name="name" type="text" placeholder="Alexandra LoremIpsum" maxlength="30" required>
+                    <input name="email" type="email" placeholder="alexandra@gmail.com" required>
```

Comment:
- [ ] The placeholder for the name input is supposed to be `full name`. What you saw on the template is just an example of a name and not a placeholder. Please try and fix that.
- [ ] The placeholder for the email input is supposed to be `email`. What you saw on the template is just an example of an email address and not a placeholder. Please try and fix that.

Reply:
Hi. I did the design for the mobile version. Should I do it for the desktop as well? because it says that it is not required. what you do is make the screen 500 px and you see the mobile version. Thanks.